Assets/Scripts/Enemy/Minion/MinionController.cs

```diff
@@ -1,18 +1,20 @@
 using System.Collections;
 using System.Collections.Generic;
+using Unity.VisualScripting;
 using UnityEngine;
 
 public class MinionController : MonoBehaviour
 {
     //Basic properties and component refs...
```

Comment:
Would you mind taking the opportunity to add headers similar to how it's setup in the BossController script?
Things like `[Header("References")]`, `[Header("Parameters")]`, whatever makes sense.